modules/auxiliary/gather/peplink_bauth_sqli.rb

```diff
@@ -196,7 +196,7 @@ def check
         'method' => 'GET',
         'cookie' => "bauth=' or #{payload}--"
       })
-      fail_with 'Unable to connect to target' unless res
+      fail_with 'Unable to connect to target' , "#{target_uri.path}" unless res
```

Comment:
Shouldn't the first argument to fail_with be a `Failure::` constant? Something like `fail_with Failure::Unreachable, 'Unable to connect to target' unless res`.

Reply:
Will update, I think I might expand the check to be a bit more explicit and do a bit more pre-validation as well.